WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -56,8 +65,11 @@ struct POSSendReceiptView: View {
                 Text(errorMessage)
                     .font(POSFontStyle.posBodyRegular)
                     .foregroundColor(.red)
+                    .padding(.bottom, Constants.errorMessagePadding)
             }
 
+            Spacer().frame(height: keyboardHeight)
```

Comment:
Perhaps you could make a `viewModifier` which you use on this spacer, so we don't declare all this keyboard observation twice?
```suggestion
            Spacer()
                .keyboardResponsiveHeight(
                    withLargeKeyboard: Constants.keyboardShownButtonSpacing, 
                    withoutKeyboard: Constants.keyboardHiddenButtonSpacing)
```